test/e2e/selinux_mount_readiness.go

```
@@ -0,0 +1,616 @@
+			if err != nil {
+				g.Fail(fmt.Sprintf("Failed to get kubeconfig: %v", err))
+			}
+			agentConfig := rest.AddUserAgent(kubeConfig, clientName)
+			kubeClient = kubernetes.NewForConfigOrDie(agentConfig)
+			configClient = cfgclientset.NewForConfigOrDie(agentConfig)
+			opClient = opclient.NewForConfigOrDie(agentConfig)
+			dynClient = dynamic.NewForConfigOrDie(agentConfig)
+
+			// Use a standalone context: Ginkgo cancels the context passed to
+			// BeforeEach when the hook returns, before the It block runs.
+			testContext, cancel = context.WithTimeout(context.Background(), testTimeout)
```

golangci-lint (2.12.2)

[error] nested context in function literal

(fatcontext)